### DISCUSSION TITLE
Tag a new release with the new updates please

### DISCUSSION THREAD

**Discussion by goellner** [statamic-rad-pack/meilisearch#14] [Help]
Could you tag a new release to support statamic 4, laravel 10 and meilisearch 1.2 ? 

I saw that the PR already got merged, just need a release now

Thank you!

**Comment by duncanmcclean** [comment]
Closing as I believe this was done in [2.0.0](https://github.com/statamic-rad-pack/mellisearch/releases/tag/2.0.0).
